openedx/core/djangoapps/edx_global_analytics/tests/test_utils.py

```diff
@@ -4,16 +4,24 @@
 
 import datetime
 
+from certificates.models import GeneratedCertificate
```

Comment:
Wrong sorting imports. 
Please read the edx style guide: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines

As a way you can use 'isort' in the terminal on your local env.